src/sage/schemes/elliptic_curves/ell_field.py

```diff
@@ -1529,6 +1533,15 @@ def isogenies_prime_degree(self, l=None, max_l=31):
             sage: E.isogenies_prime_degree(5)
             []
 
+        The optional argument `cyclic_after` can be used to easily construct cyclic
```

Comment:
```suggestion
        The optional argument ``cyclic_after`` can be used to easily construct cyclic
```
In docstrings, single backticks means $\LaTeX$ code, double backticks means code.